build.gradle

```diff
@@ -93,9 +93,10 @@ subprojects {
         }
     }
 
-    plugins.withId(rootProject.libs.plugins.hilt.get().pluginId) {
-        hilt.enableAggregatingTask = true
-    }
+    // Disable this whilst trying out napt #951
+    // plugins.withId(rootProject.libs.plugins.hilt.get().pluginId) {
+    //     hilt.enableAggregatingTask = true
```

Comment:
it appears to be [enabled by default](https://github.com/google/dagger/blob/master/java/dagger/hilt/android/plugin/main/src/main/kotlin/dagger/hilt/android/plugin/HiltExtension.kt)

Reply:
Ah that must have changed 🤔